File: 0two-sum-ii-input-array-is-sorted/0two-sum-ii-input-array-is-sorted.js
const twoSum = (numbers, target) => {
    let left = 0, right = numbers.length - 1;
    while (left < right) {
        let curr_sum = numbers[left] + numbers[right];
        if (curr_sum === target) {
            return [left + 1, right + 1];
        } else if (curr_sum < target) {
            left++;
        } else {
            right--;
        }
    }
    return [];
};

File: 0two-sum-ii-input-array-is-sorted/0two-sum-ii-input-array-is-sorted.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int left = 0, right = numbers.length - 1;
        while (left < right) {
            int curr_sum = numbers[left] + numbers[right];
            if (curr_sum == target) {
                return new int[] {left + 1, right + 1};
            } else if (curr_sum < target) {
                left++;
            } else {
                right--;
            }
        }
        return new int[] {};
    }
}